packages/design-system/vue/cypress/support/component.ts

```diff
@@ -39,7 +41,12 @@ declare global {
   }
 }
 
-Cypress.Commands.add('mount', mount);
+// Use this to mount without ActionBar wrapper component
+// Cypress.Commands.add('mount', mount);
+
+// Augment mount command with an ActionBar wrapper component
+// The ActionBar is where we place app-level settings like theme
+Cypress.Commands.add('mount', (component, ...args) => mount(() => h(ActionBar, {}, () => h(component, args[0].props, args[0].slots)), ...args));
```

Comment:
Presumably the props and slots should be omitted from the args passed to mount?

Also this is getting a bit dense and hard to parse, I'd suggest splitting this to be multiline